platform/android/export/export_plugin.cpp

```diff
@@ -3302,10 +3307,6 @@ Error EditorExportPlatformAndroid::export_project_helper(const Ref<EditorExportP
 	zipClose(unaligned_apk, nullptr);
 	unzClose(pkg);
 
-	if (err != OK) {
-		CLEANUP_AND_RETURN(err);
-	}
```

Comment:
Doesn't seem to be necessary, we never reassign `err` since the last check above.